src/ansys/pre_commit_hooks/assets/licenses.json

```diff
@@ -0,0 +1,630 @@
+{
```

Comment:
Can't we pull this whenever needed? Is it that time consuming that we have to keep it in our repo?

Reply:
I'm sure this file is coming from an external source. We should perform a GET request to retrieve it whenever needed instead of hosting it ourselves